controller/api/destination/watcher/ip_watcher.go

```diff
@@ -5,10 +5,10 @@ import (
 	"fmt"
 	"sync"
 
+	"github.com/gogo/status"
```

Comment:
Why the switch from grpc to gogo?

Reply:
+1 it would be best to avoid adding a new dependency if we can avoid it. (I believe i've seen vscode just magically do this sort of thing in the past?)

Reply:
Ah sorry yea this snuck in from a VS Code auto import or something